File: app/src/main/java/com/chooloo/www/callmanager/ui/main/MainMvpView.java
package com.chooloo.www.callmanager.ui.main;

import android.content.Intent;

import androidx.fragment.app.Fragment;

import com.chooloo.www.callmanager.ui.base.MvpView;

public interface MainMvpView extends MvpView {
    void onSetup();

    int getCurrentPosition();

    void checkIncomingIntent();

    void showDialpad(boolean isShow);

    void setDialNumber(String number);

    String getDialNumber();

    void goToSettings();

    void goToAbout();

    void showMenu(boolean isShow);

    void handleViewIntent(Intent intent);
}

File: app/src/main/java/com/chooloo/www/callmanager/ui/main/MainMvpView.kt
package com.chooloo.www.callmanager.ui.main

import com.chooloo.www.callmanager.ui.base.MvpView

interface MainMvpView : MvpView {
    var dialpadNumber: String
    val isDialpadShown: Boolean

    fun showDialpad(isShow: Boolean)
    fun showMenu(isShow: Boolean)
    fun goToSettings()
    fun goToAbout()
}